packages/flutter_tools/lib/src/commands/build.dart

```diff
@@ -8,12 +8,14 @@ import 'dart:io';
 import '../globals.dart';
 import '../runner/flutter_command.dart';
 import 'build_apk.dart';
+import 'build_ios.dart';
 import 'build_flx.dart';
 
 class BuildCommand extends FlutterCommand {
   BuildCommand() {
     addSubcommand(new BuildApkCommand());
     addSubcommand(new BuildCleanCommand());
+    addSubcommand(new BuildIOSCommand());
```

Comment:
Is this to build ipas? Perhaps call this `ipa` for parity with the `apk` command. Or we could rename both to `android` and `ios`. In any case, there should be some similarity.